.env.sample

```diff
@@ -110,4 +110,6 @@ REACT_APP_HIDDEN_MAP_LEGEND_ITEMS='Partially Sprayed'
 # jurisdiction metadata config identifiers
 REACT_APP_JURISDICTION_METADATA_RISK=jurisdiction_metadata-risk
 REACT_APP_JURISDICTION_METADATA_RISK_PERCENTAGE=90
+# plan status to hide on Manage plans and reporting pages separated by comma.
+REACT_APP_HIDDEN_PLAN_STATUSES='active,draft'
```

Comment:
This should be only `retired` by default, no?

Reply:
Yah it should be `retired`